src/cargo/ops/cargo_install.rs

```diff
@@ -745,7 +747,7 @@ pub fn install(
         let path = gctx.get_env_os("PATH").unwrap_or_default();
         let dst_in_path = env::split_paths(&path).any(|path| path == dst);
 
-        if !dst_in_path {
+        if !dst_in_path && !dry_run {
```

Comment:
I'm mixed about whether we should show this in dry-run or not.

Reply:
Fixed in 86415010617b63c981fef8c18328331e11d47434.
I added it in the first place because i was missing information about the executable, notably the path, so he was not relevant. Since now we know where the binary will be located this waning seems relevant to me.